### PR TITLE
Link directly to each chapter's LO in README

### PR DESCRIPTION
**Pull request by tanho63**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,24 +22,24 @@ Don't forget to check out the [Learning Objectives for each week](https://r4ds.g
...
+- **6: How do I make sure my package works?** Chapters [12 ("Testing")]() & [19 ("Automated checking")]()
+- **7: How do I describe my package?** Chapters [8 ("Package metadata")]() & [9 ("Licensing")]()
+- **8: How do I include things like data or other files?** Chapters [14 ("External data")]() & [16 ("Installed files")]()
+- **9: How do I use compiled code and other obscure package-y things?** Chapters [15 ("Compiled code")]() & [17 ("Other components")]()
+- **10: How do I release my package?** Chapter [20 ("Releasing a package")]()
 
 ### Presentations & Recordings
 
-**1: Why should I read this book?** Chapters 1 ("Introduction") & 2 ("The whole game"); Club goals overview
+**1: Why should I read this book?** Chapters [1 ("Introduction")]() & [2 ("The whole game")](); Club goals overview
```
Comment (jonthegeek) [inline README.md]:
```suggestion
**1: Why should I read this book?** Chapters [1 ("Introduction")](https://r4ds.github.io/bookclub-rpkgs/Companion/docs/introduction.html) & [2 ("The whole game")](https://r4ds.github.io/bookclub-rpkgs/Companion/docs/the-whole-game.html); Club goals overview
```

File: README.md [hunk 2]
```diff
@@ -22,24 +22,24 @@ Don't forget to check out the [Learning Objectives for each week](https://r4ds.g
...
 
 - Cohort 1 - [Jon Harmon](https://twitter.com/jonthegeek): [slides](https://r4ds.github.io/bookclub-rpkgs/Presentations/Week01/Cohort1/chapters1-2.html#1) | [video](https://youtu.be/FR6NsbkYhcw)
 
-**2: How do I set up my system and get started?** Chapters 3 ("System setup") & 4 ("Package structure and state")
+**2: How do I set up my system and get started?** Chapters [3 ("System setup")]() & [4 ("Package structure and state")]()
```
Comment (jonthegeek) [inline README.md]:
```suggestion
**2: How do I set up my system and get started?** Chapters [3 ("System setup")](https://r4ds.github.io/bookclub-rpkgs/Companion/docs/system-setup.html) & [4 ("Package structure and state")](https://r4ds.github.io/bookclub-rpkgs/Companion/docs/package-structure-and-state.html)
```

File: README.md [hunk 3]
```diff
@@ -22,24 +22,24 @@ Don't forget to check out the [Learning Objectives for each week](https://r4ds.g
...
 
 - Cohort 1 - [Asmae Toumi](https://twitter.com/asmae_toumi): [slides](https://r4ds.github.io/bookclub-rpkgs/Presentations/Week02/Cohort01/chap3-4.html#1) | [video](https://youtu.be/3r-EPc9XqxE)
 
-**3: How should I work on my package?** Chapters 5 ("Fundamental development workflows") & 18 ("Git and GitHub")
+**3: How should I work on my package?** Chapters [5 ("Fundamental development workflows")]() & [18 ("Git and GitHub")]()
```
Comment (jonthegeek) [inline README.md]:
```suggestion
**3: How should I work on my package?** Chapters [5 ("Fundamental development workflows")](https://r4ds.github.io/bookclub-rpkgs/Companion/docs/fundamental-development-workflows.html) & [18 ("Git and GitHub")](https://r4ds.github.io/bookclub-rpkgs/Companion/docs/git-and-github.html)
```

**Review by jonthegeek — CHANGES_REQUESTED** [review]
Added the rest of the links.

**Comment by jonthegeek** [comment]
Oh ha I missed that it was still in progress!

**Comment by tanho63** [comment]
lmao thanks ya i got a phone call

**Comment by tanho63** [comment]
Hmm. Looks like you used wip in the title and it's then included in the URL. Means double work if we keep adding. I'll just make these ready for review and we can add other chapters later

**Comment by jonthegeek** [comment]
Showing WIP in the TOC requires that it be in the title. I think it's fine to only add them here as they're done.

**Comment by tanho63** [comment]
May want to delay closing this PR in favor of the other one first

**Review by jonthegeek — APPROVED** [review]
LGTM!

**Comment by jonthegeek** [comment]
Update links! Be sure to pull if you edit locally to pick up my changes.

**Review by jonthegeek — CHANGES_REQUESTED** [review]
Change links

**Review by jonthegeek — APPROVED** [review]
Woot!

**Pull request merged**